core/capabilities/ccip/ocrimpls/contract_transmitter_test.go

```diff
@@ -588,6 +590,24 @@ type TestGasEstimatorConfig struct {
 	bumpThreshold uint64
 }
 
+func (g *TestGasEstimatorConfig) DAOracle() evmconfig.DAOracle {
+	return &TestDAOracleConfig{}
+}
+
+type TestDAOracleConfig struct {
+	evmconfig.DAOracle
+}
+
+func (d *TestDAOracleConfig) OracleType() toml.OracleType { return toml.OPStack }
+func (d *TestDAOracleConfig) OracleAddress() *types.EIP55Address {
+	a, err := types.NewEIP55Address("0x420000000000000000000000000000000000000F")
+	if err != nil {
+		panic(err)
+	}
+	return &a
+}
```

Comment:
This looks eth specific but the capability generally tries to keep all the chain specific things separated. Might be worth discussing /w @asoliman92  at some point